spacewalk/config/spacewalk-config.spec

```diff
@@ -120,34 +119,34 @@ mkdir -p $RPM_BUILD_ROOT/etc/pki/tls/private/
-sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES headers
-sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES xsendfile
-sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES filter
-sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES deflate
-sysconf_addword /etc/sysconfig/apache2 APACHE_SERVER_FLAGS SSL
-sysconf_addword /etc/sysconfig/apache2 APACHE_SERVER_FLAGS ISSUSE
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES version
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES proxy
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES proxy_ajp
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES proxy_wstunnel
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES rewrite
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES headers
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES xsendfile
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES filter
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_MODULES deflate
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_SERVER_FLAGS SSL
+sysconf_addword %{_sysconfdir}/sysconfig/apache2 APACHE_SERVER_FLAGS ISSUSE
 %endif
 
 ### TO-REMOVE AFTER: 2023-12-01
-if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task.SSHMinionActionExecutor.parallel_threads[[:space:]]*=" /etc/rhn/rhn.conf >/dev/null; then
-    sed -i "s/taskomatic.com.redhat.rhn.taskomatic.task.SSHMinionActionExecutor.parallel_threads[[:space:]]*=\(.\+\)/taskomatic.sshminion_action_executor.parallel_threads =\1/" /etc/rhn/rhn.conf
+if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task.SSHMinionActionExecutor.parallel_threads[[:space:]]*=" %{_sysconfdir}/rhn/rhn.conf >/dev/null; then
+    sed -i "s/taskomatic.com.redhat.rhn.taskomatic.task.SSHMinionActionExecutor.parallel_threads[[:space:]]*=\(.\+\)/taskomatic.sshminion_action_executor.parallel_threads =\1/" %{_sysconfdir}/rhn/rhn.conf
 fi
-if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task.MinionActionExecutor.parallel_threads[[:space:]]*=" /etc/rhn/rhn.conf >/dev/null; then
-    sed -i "s/taskomatic.com.redhat.rhn.taskomatic.task.MinionActionExecutor.parallel_threads[[:space:]]*=\(.\+\)/taskomatic.minion_action_executor.parallel_threads =\1/" /etc/rhn/rhn.conf
+if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task.MinionActionExecutor.parallel_threads[[:space:]]*=" %{_sysconfdir}/rhn/rhn.conf >/dev/null; then
+    sed -i "s/taskomatic.com.redhat.rhn.taskomatic.task.MinionActionExecutor.parallel_threads[[:space:]]*=\(.\+\)/taskomatic.minion_action_executor.parallel_threads =\1/" %{_sysconfdir}/rhn/rhn.conf
 fi
-if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task" /etc/rhn/rhn.conf >/dev/null; then
+if egrep -m1 "^taskomatic.com.redhat.rhn.taskomatic.task" %{_sysconfdir}/rhn/rhn.conf >/dev/null; then
     echo "WARNING: Found deprecated configuration items in /etc/rhn/rhn.conf"
 fi
 ### END
```

Comment:
nitpick: Maybe we should just do what the comment say instead of fixing the syntax?

Reply:
removed the full block. I wasn't sure that it was "safe to drop it" even  if the  date  in the  comment was in the past.